AuctionHouse/Overview.md

```diff
@@ -0,0 +1,129 @@
+    
+## Auctioning Fungible Assets
+So far, we've talked about exchanging assets using an Auction House account, but we've not dug into what type of assets can be exchanged that way. The most popular assets that can be listed in an Auction House are [Non-Fungible Tokens (NFTs)](/resources/definitions#non-fungible-tokens).
+
+However, these are not the only assets that can benefit from the Auction House program. In fact, an asset can be any SPL Token so long as it has a Metadata account attached to its Mint account. If you'd like to know more about SPL Token and Metadata accounts, you can [read more about this in the overview of our Token Metadata program](/programs/token-metadata/overview).
+    
+## Buying asset using a custom SPL Token
+In all the examples showcased above, we use SOL as the exchange currency to discuss how the Auction House program works. But SOL is not the only currency that can be configured to be used for exchanging assets and the Auction House program allows the marketplaces to configure any SPL-token to act as the currency for exchanging assets on their platform.
+
+This can be achieved by setting the `treasuryMint` parameter in the Auction House account to the mint account of the SPL-token of your liking.
+
+## Custom Order Matching
+When we were discussing Trade States, there was a third Trade State which was shown in the Trade State diagram: the `FreeSellerTradeState`. What's the utility of this Trade State?
+
+During the introduction to the Auction House program, we briefly discussed how Auction House can be used by marketplaces to provide their own custom logic on how assets should be exchanged. This is where the `FreeSellerTradeState` comes in.
+
+When a buyer intentionally lists their asset for a price of 0 SOL / SPL-tokens, the `FreeSellerTradeState` is generated. The Auction House can then change the sale price to match a matching bid that is greater than 0. This allows the Auction House to do complicated order matching to find the best price for the seller and the marketplaces can write their own custom logic to do this order matching.
+
+## Auctioneer
+In this page, we have discussed a lot about auctions and even went through a number of examples showcasing these auctions. But there was something common in all of these auctions: these auctions are all [**Double Auctions**](https://blogs.cornell.edu/info2040/2021/11/29/four-common-types-of-auctions/#:~:text=sealed%2Dbid%20auction.-,DOUBLE%20AUCTION,-Both%20buyers%20and): un-timed auctions where the buyers and sellers, bid and list until they find a common ground. There are several other forms of auctions such as English auctions and Dutch auctions which are Timed auctions that have become popular in the Solana ecosystem.
+The current Auction House implementation is designed with instant sales in mind and currently has no features that enable other auctions types.
```

Comment:
```suggestion
The Auction House implementation is purposefully designed with instant sales in mind and does not offer other auction types out-of-the-box.
```

Note that we're making AH simple by design, so Auctioneer programs can have as much power as possible.